x-pack/plugins/ml/server/routes/schemas/anomaly_detectors_schema.ts

```diff
@@ -183,4 +183,18 @@ export const getCategoriesSchema = schema.object({
   jobId: schema.string(),
 });
 
+export const getModelSnapshotsSchema = schema.object({
+  /** Snapshot id */
```

Comment:
Nit - I think these end up in the generated API docs, in which case we normally use uppercase ID. Same for job ID below.